acceptance/tests/facts/windows.rb

```diff
@@ -22,7 +22,10 @@
     os_version  = '2012 R2'
     kernel_version = '6.3'
   elsif agent['platform'] =~ /-10/
-    os_version  = /^10\./
+    os_version  = '10'
```

Comment:
@Magisus I think there are a couple bugs in the code prior to this PR and I want to confirm if you were looking to fix both of them?

The first is that the `os.release.major` and `os.release.full` are reporting the same version string on consumer windows 10, i.e. this output on Windows 10 running facter 3.x:
```
os => {
  architecture => "x64",
  family => "windows",
  hardware => "x86_64",
  name => "windows",
  release => {
    full => "10.0.10586",
    major => "10.0.10586"
  },
...
```
whereas the output should actually be something like:
```
os => {
  architecture => "x64",
  family => "windows",
  hardware => "x86_64",
  name => "windows",
  release => {
    full => "10.0.10586",
    major => "10"
  },
...
```

The second bug is that on Windows Server 2016, the `os.release.full` and `os.release.major` facts return the kernel version string, ie `10.0.10586`, rather than the string `2016` as it would have done under Facter 2.x.

Is this PR intended to address both of these, or just one or the other? 

I noticed this because down below on lines 42, 43 this facter test it appears to assert that `os.release.full` and `os.release.major` are the same string, which shouldn't be the case for Windows 10 consumer edition.